ui/components/src/utils/theme-gen.ts

```diff
@@ -39,7 +39,21 @@ export function generateChartsTheme(muiTheme: MuiTheme, echartsTheme: EChartsThe
       left: 20,
       containLabel: true,
     },
-    color: ['#8dd3c7', '#fb8072', '#80b1d3', '#bebada', '#fdb462'],
+    // Accessible categorical palette from: https://colorbrewer2.org/#type=qualitative&scheme=Paired&n=12
+    color: [
```

Comment:
These static accessible colors are used for alternate `visual.palette.kind: 'Categorical'` and the default `visual.palette.kind: 'Auto'` is still generated from series names (so they are consistent across panels and can work with many series so colors don't repeat)